### PR TITLE
#1181 - Add another test

### LINKED ISSUE #1181: Add new Behat context to do REST result checks
We need to be able to assert text etc. to be present in JSON outputs, e.g. like was attempted in https://github.com/comicrelief/campaign/pull/1177/commits/da18597fa651e60b2e0739a4f5dbd9314bf1bdab

We can probably follow this approach: http://keithloy.me/testing-rest-api-with-behat/

### PR DESCRIPTION
**Pull request by Saphyel**
Fixes #1181 

## Changes proposed in this PR

- Add a generic one

### CODE REVIEW

File: tests/behat/features/rest.feature [hunk 1]
```diff
@@ -4,3 +4,4 @@ Feature: REST API
   Scenario: Check main menu REST endpoint
     When I do a GET request to "/entity/menu/main/tree"
     Then I should find in the position 1 of the menu the "title" with the value "Fundraise (Landing)"
+    Then I want to find the text "FAQ" in the JSON
```
Comment (pvhee) [inline tests/behat/features/rest.feature]: Can we make this the same as non-JSON syntax? i.e. `And I should see "FAQ"`

Comment (Saphyel) [reply tests/behat/features/rest.feature]: Nop, is already taken.

Comment (pvhee) [reply tests/behat/features/rest.feature]: What about `"And I should see XXX in the JSON" ?`

**Comment by Saphyel** [comment]
duplicated of #1212

**Pull request closed without merging**